utils/libnetwork/Makefile

```diff
@@ -1,7 +1,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libnetwork
-PKG_VERSION:=0.8.0-dev.2
+PKG_SOURCE_VERSION:=e7933d41
+PKG_VERSION:=0.8.0-dev.2-${PKG_SOURCE_VERSION}
```

Comment:
Change the layout of the Makefile to something like this: https://github.com/openwrt/packages/blob/master/net/cifsd-tools/Makefile

Basically, use PKG_SOURCE_DATE and PKG_SOURCE_VERSION instead of PKG_VERSION.

Reply:
Using `PKG_SOURCE_VERSION` and `PKG_SOURCE_DATE`